### ISSUE TITLE
Qt Send tab: we should rename InvoiceList from "Outgoing payments" - it confuses users

### ISSUE THREAD

**Issue by SomberNight** [spesmilo/electrum#6711]
Currently, in the Send tab, the list of invoices at the bottom is labelled ["Outgoing payments"](https://github.com/spesmilo/electrum/blob/1c07777e135d28fffa157019f90ccdaa002b614e/electrum/gui/qt/main_window.py#L1394). Seems to me this is confusing for some newbies, who sometimes think that there being a "pending" invoice in this list means having already broadcast an unconfirmed transaction.

In general I have seen multiple occasions where users were looking at the Send tab instead of the History tab.

<details>
 <summary>pic of Send tab in 4.0.4</summary>

![qt_send_tab_4_0_4](https://user-images.githubusercontent.com/29142493/98189345-b2841000-1f0c-11eb-8d86-1f3ab7cf04ed.PNG)
</details>

I think it might help if we tried to come up with a less confusing label for the invoices list. Similarly we should probably rename the list in the Receive tab for the other direction (currently named ["Incoming payments"](https://github.com/spesmilo/electrum/blob/1c07777e135d28fffa157019f90ccdaa002b614e/electrum/gui/qt/main_window.py#L1142)).

In 3.3.8, these lists were called ["Invoices"](https://github.com/spesmilo/electrum/blob/e81f4bdcd11a072e7c4f38fb1c7eec19c2f7e1a8/electrum/gui/qt/main_window.py#L1386) and ["Requests"](https://github.com/spesmilo/electrum/blob/e81f4bdcd11a072e7c4f38fb1c7eec19c2f7e1a8/electrum/gui/qt/main_window.py#L964) respectively.

How about changing the labels to "Invoices" (in Send tab), and "Receive requests" (in Receive tab)?

**Comment by SomberNight** [comment]
Sort of related, maybe we should consider changing the status string from "Pending" to "Unpaid":
https://github.com/spesmilo/electrum/blob/1c07777e135d28fffa157019f90ccdaa002b614e/electrum/invoices.py#L44

However, this string is exposed to CLI/RPC users so it would be a breaking change.

**Comment by ecdsa** [comment]
"invoices" and "requests" are confusing too...
I agree with changing 'pending' to 'unpaid'. maybe that would be enough to fix this issue.

**Comment by yogi-lo** [comment]
Newbie here... using ver 4.04...I feel the terms "incoming payments" on the receive Tab and "outgoing payments" on the Send tab are clear.

The confusion (for me) is in the "status" columns. Super confused about why sats received on October 22 and 28 that had a green check next to them are now showing as status "pending" clock with an exclamation (in Incoming payments list).

On History tab, those same transactions are showing as confirmed with green check. 

Also I had issues sending to cold storage. It would be cleaner to have 1 send screen with all the parameters (i.e., instead of having to click "send" then the "advanced tab" to change fees, then click broadcast).

To a newbie, "send" means "broadcast". I knew fees were high and wanted to adjust them manually, but was stumped as to how to get to that slider screen. So reluctantly hit "send" (thinking I was broadcasting) and then I saw the "advanced" tab.  If all the parameters cannot be on one page, perhaps use the label "next" or "more" instead of "send". 
 
When I did get to the page with the fee slider, it froze; then it  warned the fee was too high (mining fees were high at the time). Then it set the fee to zero and I tried to slide the fee again and it said invalid transaction...and then it said "sent". It stayed in "pending" status for 2 days. Then I right clicked and deleted.  I tried sending again and it was "unconfirmed" in the history tab, so I thought all was well. Then 1 day later the history tab said that the transaction was "local" and needed to be broadcast. lol.  I thought I broadcast it?

So i clicked "broadcast" and flashed that it was "sent"...and now it says "unconfirmed [rbf,4.2 sat/b, 5.37 MB] ...which looks like the other unconfirmed message. I'll check in on it again tomorrow.

Feel free to reach out if you want newbie feedback on anything ...I want to become more proficient in this.  

Appreciate all the work that goes into this project.

Many thanks for your time and patience.
